_posts/de/newsletters/2022-11-02-newsletter.md

```diff
@@ -0,0 +1,277 @@
+  einer Transaktion über den [CPFP][topic cpfp]-Mechanismus Gebühren
+  hinzuzufügen, die für das LN-Zweiparteien-Vertragsprotokoll nicht
+  [angreifbar][topic transaction pinning] wären. Teinturier
+  [merkt jedoch an][bolts #845], dass die Verwendung von CPFP voraussetzt, dass
+  jeder LN-Knoten einen Pool von Nicht-LN-UTXOs bereithält, die jederzeit
+  ausgegeben werden können. Im Vergleich dazu ermöglicht das Vorsignieren
+  mehrerer Versionen von HTLCs mit jeweils unterschiedlichen Gebühren, dass
+  diese Gebühren direkt aus dem Wert des HTLC bezahlt werden können ---
+  abgesehen von Fällen, in denen keine der vorgegebenen Gebühren hoch genug war,
+  ist keine zusätzliche UTXO-Verwaltung erforderlich.
+
+    Er bittet andere LN-Entwickler um Unterstützung für die Idee, HTLCs mit
+    mehreren Gebührenraten anzubieten. Zum Zeitpunkt der Erstellung dieses
+    Artikels wurde die gesamte Diskussion auf [Teinturiers PR][bolts #1036]
+    geführt.
+
+## Releases und Release-Kandidaten
+
+*Neue Releases und Release-Kandidaten für beliebte Bitcoin
+Infrastrukturprojekte. Bitte erwäge auf die neuen Versionen
+zu wechseln oder beim Testen von Release-Kandidaten auszuhelfen.*
+
+- [LND 0.15.4-beta][] und [0.14.4-beta][lnd 0.14.4-beta] sind
+  **sicherheitskritische** Veröffentlichungen, die Fehlerbehebungen für ein
+  Problem bei der Verarbeitung neuer Blöcke enthalten. Alle Benutzer sollten ein
+  Upgrade durchführen.
+
+- [Bitcoin Core 24.0 RC2][] ist ein Release Candidate für die nächste Version
+  der am weitesten verbreiteten Full-Node-Implementierung des Netzwerks. Eine
+  [Anleitung zum Testen][bcc testing] ist verfügbar.
+
+  **Warnung:** Dieser Release Candidate enthält die Konfigurationsoption
+  `mempoolfullrbf`, von der mehrere Protokoll- und Anwendungsentwickler glauben,
+  dass sie zu Problemen für Händlerdienste führen könnte; wie in den Newslettern
+  [#222][news222 rbf] and [#223][news223 rbf] beschrieben. Optech fordert alle
+  Dienste auf, die davon betroffen sein könnten, den RC zu evaluieren und sich
+  an der öffentlichen Diskussion zu beteiligen.
+
+## Nennenswerte Code- und Dokumentationsänderungen
+
+*Erwähnenswerte Änderungen diese Woche in [Bitcoin Core][bitcoin core repo],
+[Core Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], und [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #23927][] schränkt `getblockfrompeer` auf beschnittenen (pruned)
+  Knoten auf Blockhöhen unterhalb des aktuellen Synchronisationsfortschritts des
+  Knotens ein. Dies verhindert, dass ein Footgun entsteht, der zukünftige Blöcke
+  abruft und damit die Blockdateien des Knotens für das Pruning ungeeignet macht.
```

Comment:
_beschnitten_ is suboptimal for pruning, but I couldn't fine a better German term. As a noun, I would suggest to keep _Pruning_.

Reply:
Auf den zweiten Blick, vielleicht bleiben wir einfach dabei: 
![image](https://user-images.githubusercontent.com/4060799/202030370-dc639784-1bac-4a48-9093-6b68c3248893.png)